pkg/reports/reports_test.go

```diff
@@ -14,17 +14,19 @@ import (
 )
 
 var fixtureReport1 = api.Report{
-	ID:               "fixtureReport1",
-	PreflightVersion: "version.PreflightVersion",
-	Package:          "examplePackage.ID",
-	PackageInformation: api.PackageInformation{
-		Namespace: "examplePackage.Namespace",
-		ID:        "examplePackage.ID",
-		Version:   "examplePackage.PackageVersion",
+	ReportMetadata: api.ReportMetadata{
+		ID:               "fixtureReport1",
+		PreflightVersion: "version.PreflightVersion",
+		Package:          "examplePackage.ID",
+		PackageInformation: api.PackageInformation{
+			Namespace: "examplePackage.Namespace",
+			ID:        "examplePackage.ID",
+			Version:   "examplePackage.PackageVersion",
+		},
+		Cluster: "exampleCluster",
 	},
 	Name:        "examplePackage.Name",
 	Description: "examplePackage.Description",
```

Comment:
I think I prefer #115 because as you say it's hard to decide what is and isn't metadata. e.g. for me, I'd expect name and ID to be at the same level.